documentation/library_guide/introduction.rst

```diff
@@ -65,6 +65,8 @@ Difference with Standard C++ Parallel Algorithms
 
 * oneDPL execution policies only result in parallel execution if random access iterators are provided,
   the execution will remain serial for other iterator types.
+* Function object parameters passed in to algorithms executed with device policies must provide const function call
```

Comment:
```suggestion
* Function objects passed in to algorithms executed with device policies much provide ``const``-qualified ``operator()``.
```

Reply:
"much" should be "must", otherwise it looks good to me.

Reply:
Thanks, done.